models/stg_linkedin.yml

```diff
@@ -369,4 +369,97 @@ models:
           ENTERPRISE - This value cannot be used to create accounts through the API and is reserved for accounts created by LinkedIn's internal ad operations systems.
 
       - name: is_latest_version
-        description: Boolean of whether the record is the latest version of the account.
\ No newline at end of file
+        description: Boolean of whether the record is the latest version of the account.
+
+  - name: stg_linkedin_ads__geo
+    description: Contains geographic data for LinkedIn Ads, including IDs and names of locations.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation
+            - geo_id
+    columns:
+      - name: geo_id
+        description: Unique identifier for the geographic entity, used for joins in reporting tables.
+        tests: 
+          - not_null
+      - name: value
+        description: Name of the geographic entity.
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+
+  - name: stg_linkedin_ads__monthly_ad_analytics_by_country
+    description: Monthly LinkedIn Ads performance metrics aggregated by member country and campaign.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation
+            - campaign_id
+            - date_month
+            - member_country_geo_id
+    columns:
+      - name: campaign_id
+        description: Identifier of the LinkedIn Ads campaign.
+      - name: date_month
+        description: Month of the reporting data, aliased as `date_month`.
+        tests: 
+          - not_null
+      - name: clicks
+        description: The count of chargeable clicks.
+      - name: member_country_geo_id
+        description: ID representing the country of the LinkedIn members who interacted with the campaign.
+      - name: impressions
+        description: This is the count of "impressions" for Direct Ads and Sponsored Updates and "sends" for InMails.
+      - name: conversion_value_in_local_currency
+        description: The value generated by your conversions, displayed in your LinkedIn account's local currency.
+      - name: cost
+        description: The cost of the ads in the local currency or USD.
+      - name: external_website_conversions
+        description: Number of conversions that occurred on an external website as a result of the ad.
+      - name: one_click_leads
+        description: Number of one-click leads generated from the ad campaign.
+      - name: external_website_conversions
+        description: The actions taken on your website that you've defined as valuable to your business after clicking on LinkedIn ads.
+      - name: one_click_leads
+        description: Leads submitted after clicking on LinkedIn ads.
```

Comment:
These columns are duplicated. I think the second set of columns have better descriptions, so I'd opt to keep those, but will leave up to you!

Reply:
Unsure how this happened. Removed.